website/interview_exp/api/views.py

```diff
@@ -47,13 +47,15 @@ class IEListView(ListCreateAPIView):
     ordering_fields = ['updated_at', 'total_Compensation', 'year']
 
     def get_queryset(self):
+        print('yes')
```

Comment:
remove this print statement.
`good to have : try using logging package and use logger.info and logger.error whenever needed `

Reply:
Removed print for now will add logger later